src/main/java/com/coreos/jetcd/EtcdWatch.java

```diff
@@ -0,0 +1,32 @@
+package com.coreos.jetcd;
+
+import com.coreos.jetcd.options.WatchOption;
+import com.coreos.jetcd.util.ListenableSetFuture;
+import com.coreos.jetcd.watch.Watcher;
+import com.google.protobuf.ByteString;
```

Comment:
One concern to use Java String directly is a semantic gap between Golang's string and Java's string. IIUC, etcd's key can contain illegal String for java String since Golang's string is just a wrapper of byte[]. As a result, it can happen to fail to decode as Java's string. It's not trivial to handle them correctly at jetcd layer. My suggestion is to create primitive APIs which accept/return byte[], and create utility API which convert byte[] in primitive APIs to Java String on the top of them.